exasol/toolbox/tools/lint.py

```diff
@@ -0,0 +1,80 @@
+import json
+from collections.abc import Iterable
+from dataclasses import dataclass
+from inspect import cleandoc
+from pathlib import Path
+
+import typer
+
+CLI = typer.Typer()
+
+
+@dataclass(frozen=True)
+class LintIssue:
```

Comment:
You could name it just `Finding` or `Issue` as the full path should identify which kind of finding it is. Try not to necessarily duplicate information on each hierarchy level.

```python
from exasol.toolbox.tools.lint import LintIssue

issue = LintIssue(...)
```

vs

```python
from exasol.toolbox.tools import lint

finding = lint.Finding(...)
````